spec/integration/api_request_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe "API requests for content items", :type => :request do
```

Comment:
Ah, maybe this would be better named `public_api_request_spec.rb`, i.e. it's for specs covering the public-facing `/api` endpoint. The one spec within replaces one the two tests mentioned in the other comment above (`"includes the public API URL when requested through the public endpoint"`). The other one was lost in translation, I'll re-instate it somewhere.